### DISCUSSION TITLE
Some suggestions for Options Search queries

### DISCUSSION THREAD

**Discussion by ItsJustSomeDude** [ppy/osu#22366] [Ideas] [answered]
I may be totally over reacting with this discussion.  It could be 100% user error on my part.

Whenever I open the Options menu I will always use Search immediately to find what I'm looking for.  There are some things that I think should have keywords added to them so that they will be found when searched.

"Keyb" brings up "Shortcut and Gameplay Bindings", but if I type "Keybo..." because I want to change settings about the **Keyboard**, suddenly it's gone.  Also, sometimes I type "Control", which also brings up nothing, but should bring this up.

Typing "First" does not bring up "Run Setup Wizard", even though said wizard is called "First-run setup" once you open it.  Also, now that all the Import from Stable options are gone and we have to use the First-run setup wizard for that, typing "Import" should also bring up "Run Setup Wizard".

If I want to change **Touchscreen** settings, but type "Touchs..." there are no results because they're called "Touch"

This point is an opinion, but usually when I want to "Open osu! Folder" I am looking for logs, but typing "Logs" does not pull that up.

These are only the ones that I want to change somewhat often and leave my searches empty.

**Comment by peppy** [comment, marked as answer]
Thanks, I've applied all your feedback in https://github.com/ppy/osu/pull/22371.

But I'm not sure about the "touch" settings. Do you mean the single setting in gameplay "Show gameplay cursor during touch input"?

**Reply by ItsJustSomeDude** [reply]
I was thinking "Touchscreen" would also find the main Touch toggle in the Input section.

**Reply by peppy** [reply]
Hmm, I don't even see that. Which platform are you on to see that?

**Reply by Walavouchey** [reply]
on plain-old windows (10, with no touch display) it's visible under "input", below the tablet settings

![image](https://user-images.githubusercontent.com/36758269/214150926-cee0fd56-d4f5-47b3-b7fa-bdfd24cf2d8f.png)

... shouldn't it be visible on all desktop platforms? some desktop devices have touchscreens (and i happen to have one)

**Reply by peppy** [reply]
Thanks, have addressed in https://github.com/ppy/osu/pull/22385.
